cmd/pg/backup_fetch.go

```diff
@@ -40,16 +51,43 @@ var backupFetchCmd = &cobra.Command{
 			pgFetcher = internal.GetPgFetcherOld(args[0], fileMask, restoreSpec)
 		}
 
-		internal.HandleBackupFetch(folder, args[1], pgFetcher)
+		internal.HandleBackupFetch(folder, targetBackupSelector, pgFetcher)
 	},
 }
 
+// create the BackupSelector to select the backup to fetch
+func createTargetBackupSelector(cmd *cobra.Command, args []string) (internal.BackupSelector, error) {
+	switch {
+	case len(args) == 2 && targetUserData != "":
+		fmt.Println(cmd.UsageString())
+		return nil, errors.New("Incorrect arguments. Specify backup_name OR target flag, not both.")
```

Comment:
Let's put all errors at the end of the switch block.

Reply:
fixed in 7e364e1